### ISSUE TITLE
A stable version is required to run the broker

### ISSUE THREAD

**Issue by Sun-ZhenXing** [Yakifo/amqtt#128]
Test `amqtt` broker in different system:
- Debian 11.5
- Windows 10

Broker unable to use, or does not work at all.

So I need a stable version, and it need to be submitted to pypi.

Therefore, I still use `mosquitto` as the broker and use `amqtt` as the client. I hope this community can continue to flourish.

**Comment by not-f-elsner** [comment]
Which Python version are you using?

Which error do you get?

In the meantime, please try the current master branch, instead of the latest published Pypi package

**Comment by Sun-ZhenXing** [comment]
I have tried the standard Broker commands, but have been unable to receive messages from the client. I used mosquitto as a control and mosquitto behaved normally, I used amqtt and mosquitto's clients.

I tried the main branch Broker and it performed well on my Windows, however more testing needs to be done.
- Python 3.9
- `amqtt`

As other issues have pointed out, Broker's performance may not be consistent enough.
